README.md

```diff
@@ -2,7 +2,7 @@
 
 # dotenv
 
-<img src="https://raw.githubusercontent.com/motdotla/dotenv/master/dotenv.png" alt="dotenv" align="right" />
+<img src="https://raw.githubusercontent.com/motdotla/dotenv/master/media/dotenv.svg" alt="dotenv" align="right" />
```

Comment:
make this `/dotenv.svg` rather than `/media/dotenv.svg`

Reply:
Done.